tests/performance/text_index.xml

```diff
@@ -0,0 +1,13 @@
+<test>
+  <fill_query>ALTER TABLE hits_10m_single ADD INDEX idx_sp(SearchPhrase) TYPE text(tokenizer = 'default')</fill_query>
+  <fill_query>ALTER TABLE hits_10m_single MATERIALIZE INDEX idx_sp</fill_query>
+
+  <query>SELECT count() FROM hits_10m_single WHERE hasToken(SearchPhrase, 'yandex')</query>
```

Comment:
(maybe group the queries by frequency of the search phrase?)

Reply:
ah good point, except one they are in the decreasing frequency. I fixed it and added a comment